app/Helpers/database/forum.php

```diff
@@ -317,6 +317,7 @@ function generateGameForumTopic(User $user, int $gameId): ?ForumTopicComment
 
     $topicPayload = "Official Topic Post for discussion about [game=$gameId]\n" .
         "Created " . date("j M, Y H:i") . " by [user={$user->User}]\n\n" .
+        "[b][url=https://retroachievements.org/linkedhashes.php?g=$gameId]💾 Supported Game Files[/url][/b]\n\n" .
```

Comment:
This is a redirect URL. The current URL is `route('game.hashes.index', ['game' => $gameId])`.

Thanks for asking about the emoji. For the time being I think my preference would be no emoji or iconography associated with the link.

Reply:
Oh whoops, I didn't even think about that. Let me adjust it real quick.